src/charts.js

```diff
@@ -348,6 +368,45 @@ function showTotalDownloads(sanitizedData, fromDate, toDate, showSum) {
             + dateToDayKey(toDate) + '</em>:');
 }
 
+function getDownloadData(packageNames, fromDate, toDate) {
+    return new Promise(function (accept, reject) {
+        var requestArray = [];
+        var packageNameToRequestIndex = [ ];
+        $.each(packageNames, function (index, packageName) {
+
+            var allUrls = getDownloadsUrl(packageName, fromDate, toDate);
+
+            var allDataReqs = allUrls.map(url => requestData(url));
```

Comment:
Please don't use lambdas. There's no babel in the build script and thus, you should stick with ES5.